.prettierrc.js

```diff
@@ -0,0 +1,8 @@
+module.exports = {
+    printWidth: 100,
+    tabWidth: 4,
```

Comment:
O shoot, we should make this 2, I forgot the project that I pasted this from was 4 for some reason.

Reply:
Done